instrumentation/opentelemetry-instrumentation-starlette/src/opentelemetry/instrumentation/starlette/__init__.py

```diff
@@ -186,20 +205,46 @@ def _instrument(self, **kwargs):
         _InstrumentedStarlette._client_response_hook = kwargs.get(
             "client_response_hook"
         )
+        _InstrumentedStarlette._meter_provider = kwargs.get("_meter_provider")
+
         applications.Starlette = _InstrumentedStarlette
 
+        for instance in _InstrumentedStarlette._instrumented_starlette_apps:
+            self.instrument_app(
```

Comment:
@srikanthccv 
Do we want to instrument all currently existing apps if we call `instrument()` or only apps created AFTER `instrument()` was called?

Reply:
I would expect it to instrument the apps created after the instrument is invoked.

Reply:
@rahulmukherjee68 
Might want to remove this behavior.

Reply:
Yeah @lzchen i have removed it in my latest commits